src/orca-log-catcher/OrcaLogger.res

```diff
@@ -137,8 +137,14 @@ let convertToScreamingSnakeCase = text => {
   text->String.trim->String.replaceRegExp(%re("/ /g"), "_")->String.toUpperCase
 }
 
+let toSnakeCaseWithSeparator = (str, separator) => {
```

Comment:
move this to Utils

Reply:
Cannot move this to Utils, cyclic dependency will occur